e2e_tests/tests/requirements.txt

```diff
@@ -3,12 +3,10 @@ appdirs
 pytest>=6.0.1
 pytest-timeout
 pexpect
-torch==1.4
-torchvision==0.5.0
-tensorflow==1.15.4
-# TF 1.15.4 has requirement numpy<1.19.0,>=1.16.0
-# and numpy==1.19.2 throws numpy.ufunc has the wrong size error
-numpy==1.18.5
+torch==1.7
+torchvision==0.8.1
+tensorflow==2.4.0
+numpy
```

Comment:
I doubt that we even need to specify `numpy` now.  But that's just a nit; as long as it's not pinned I don't care.